.github/workflows/publish.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - '*.*.*'
+
+jobs:
+
+  publish:
+    name: Publish package to NPM
+    runs-on: [ubuntu-latest]
+    steps:
+      - name: Use Node.js 10.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 10.x
```

Comment:
Can you also please add nodeJS 12.x
/!\ AWS Lambda now support it !! 👍

Reply:
Do you want me to bump it or add a matrix ?

Reply:
New matrix, let's keep support for 10.x too ;)

Reply:
See #4